src/payments/payments.controller.ts

```
@@ -65,4 +71,54 @@ export class PaymentsController {
   async getPaymentStatus(@Param('id', ParseUUIDPipe) paymentIntentId: string) {
     return await this.paymentService.getPaymentStatus(paymentIntentId);
   }
+
+  @Get('report/:contextId')
```

ESLint

[error] Replace `'report/:contextId'` with `"report/:contextId"`

(prettier/prettier)